src/auth/screens/auth-profile.screen.js

```diff
@@ -139,74 +148,84 @@ class AuthProfile extends Component {
 
     return (
       <ViewContainer>
-        {loading && <LoadingContainer animating={loading} center />}
-
-        {!loading &&
-          <ParallaxScroll
-            renderContent={() =>
-              <UserProfile
-                type="user"
-                initialUser={user}
-                user={user}
-                navigation={navigation}
-              />}
-            stickyTitle={user.login}
-          >
-            {user.bio &&
-              user.bio !== '' &&
-              <SectionList title="BIO">
-                <ListItem
-                  subtitle={user.bio}
-                  subtitleStyle={styles.listSubTitle}
-                  hideChevron
-                />
+        <ScrollView
+          refreshControl={
+            <RefreshControl
+              refreshing={loading}
+              onRefresh={this.refreshProfile}
+            />
+          }
+        >
+          {loading && <LoadingContainer animating={loading} center />}
+
+          {!loading &&
+            <ParallaxScroll
+              renderContent={() =>
+                <UserProfile
+                  type="user"
+                  initialUser={user}
+                  user={user}
+                  navigation={navigation}
+                />}
+              stickyTitle={user.login}
+            >
+              {user.bio &&
```

Comment:
We can use `lodash.get` to perform this kind of tests.
It should be something like `const userHasBio = get(user, ['bio'], '').length;` so you can omit the subsequent checks